tools/diamond.py

```diff
@@ -27,7 +28,9 @@ class Diamond(tools.Tool):
 
     def __init__(self, install_methods=None):
         if not install_methods:
-            install_methods = [DownloadAndBuildDiamond(URL, os.path.join(DIAMOND_DIR, 'bin', 'diamond'))]
+            install_methods = [
+                DownloadAndBuildDiamond(URL, os.path.join(DIAMOND_DIR, 'bin', 'diamond')),
+                tools.CondaPackage("diamond", version=CONDA_VERSION)]
```

Comment:
Is there a reason that Conda should only be tried if download-and-build fails? Most tools specify these installers in the opposite order for the sake of speed-optimization.